config/vufind/searches.ini

```diff
@@ -251,6 +251,10 @@ WorkKeys = year
 ;       Display catalog search results matching the terms found in the specified
 ;       GET parameter (default = "lookfor"), limited to a specified number of
 ;       matches (default = 5).  This is designed for use with non-catalog modules.
+; Databases:[result limit]:[ini name]
```

Comment:
Maybe also worth adding an example of this (commented out) to the `default_side_recommend` settings in EDS.ini, so the example there is more complete.

Reply:
Done